changelog.d/pr-9090

```diff
@@ -0,0 +1,11 @@
+synopsis: Removed mentions of "v-" from relevant documentation
```

Comment:
I’m sorry for one more ask, but in the end this patch turned out to be a non-user visible change, so it doesn’t need a changelog file. I figured that when I was thinking about the `packages` field: it doesn’t really belong to any package. So I propose we remove the changelog file altogether.

Reply:
No worries. I've removed this file.

Reply:
Good! Approving now. We'll need to find another reviewer but don't worry about that: I'll try to ping relevant folks.